File: C/String/program23.c
//Accept two string from user and concat first string after the second string

#include<stdio.h>

void strcatX(char *src, char *dest)
{
    while(*dest != '\0')
    {
        dest++;
    }

    while(*src != '\0')
    {
        *dest = *src;
        src++;
        dest++;
    }
    *dest = '\0';
}

int main()
{
    char Arr[20];
    char Brr[20];

    printf("Enter first string: \n");
    scanf("%[^'\n']s",Arr);

    printf("Enter second string: \n");
    scanf(" %[^'\n']s",Brr);

    strcatX(Arr,Brr);
    printf("String after concatination : %s ",Brr);

    return 0;
}

File: Java/String/program23.java
//Accept the string from user and copy that string into another string

import java.util.*;

class Stringx
{
    public String src;
    public String dest;

    public void Accept()
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter the string");
        src = sobj.nextLine();
    }

    public void strcpyX()
    {
        System.out.println("String after copy : ");

        dest = src;
        
        System.out.print(dest);     
    }
}

class program23
{
    public static void main(String arg[])
    {
        Stringx obj = new Stringx();

        obj.Accept();

        obj.strcpyX();
    }
}